torch/csrc/distributed/c10d/ProcessGroupNCCL.hpp

```diff
@@ -37,6 +37,11 @@
 
 namespace c10d {
 
+// Control whether to always use high priority streams
+static std::vector<std::string> TORCH_NCCL_HIGH_PRIORITY = {
+    "TORCH_NCCL_HIGH_PRIORITY",
+    "NCCL_HIGH_PRIORITY"};
```

Comment:
nit: can you limit the env name to "TORCH_NCCL_HIGH_PRIORITY" only? 
The vector format you saw for other envs are just for backward compatibility for early envs starting with "NCCL_". For new envs, we want to start with "TORCH_NCCL_".

Reply:
Done, thanks for the explanation.